tests/testecc.nim

```diff
@@ -53,7 +54,7 @@ suite "ECC/ECDSA/ECDHE tests suite":
     var expectm = """
       8ac7e464348b85d9fdfc0a81f2fdc0bbbb8ee5fb3840de6ed60ad9372e718977"""
     var secret: array[KeyLength, byte]
-    var s = keccak256.digest("ecdhAgree").data
+    var s = initPrivateKey(keccak256.digest("ecdhAgree").data)
```

Comment:
`initPrivateKey()` is a very bad name for hex conversion function, for me `initPrivateKey()` its create new private key.
First of all we will not store private keys as hexadecimal strings, and there is already present `getPrivateKey` which actually do same thing.

Reply:
It is not hex string here, `keccak256.digest("ecdhAgree").data ` is `array[32, byte]`.